schema/metrics/CommonMetrics.go

```diff
@@ -0,0 +1,48 @@
+package metrics
+
+import (
+	"fmt"
+	"reflect"
+)
+
+var LIVENESS_ARBITER = NewAggregatingMetric(
+	NewPrometheusMetric("th2_liveness", "Service liveness"),
+)
+var READINESS_ARBITER = NewAggregatingMetric(
+	NewPrometheusMetric("th2_readiness", "Service readiness"),
+)
+
+func init() {
+	RegisterLiveness("user_liveness")
+	RegisterReadiness("user_readiness")
+}
```

Comment:
Remove these preconstructed liveness/readiness metrics, user and other modules should register own metrics to follow own part of work.